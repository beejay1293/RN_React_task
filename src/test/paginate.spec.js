import React from 'react';
import { shallow } from 'enzyme';
import  Paginate from '../components/Dashboard/paginate';

describe('test Paginate component', () => {
  it('should render component', () => {
    const Page = shallow(<Paginate  />);
    expect(Page).toMatchSnapshot();
  });
});
