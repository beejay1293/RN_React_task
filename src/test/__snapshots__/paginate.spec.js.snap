// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Paginate component should render component 1`] = `
<Pagination
  aria-label="Page navigation"
  listTag="ul"
  size="sm"
  tag="nav"
>
  <PaginationItem
    disabled={true}
    id={1}
    tag="li"
  >
    <PaginationLink
      first={true}
      id={1}
      tag="a"
    />
  </PaginationItem>
  <PaginationItem
    disabled={true}
    id="prev"
    tag="li"
  >
    <PaginationLink
      id="prev"
      previous={true}
      tag="a"
    />
  </PaginationItem>
  <PaginationItem
    className=""
    id={1}
    key="0"
    tag="li"
  >
    <PaginationLink
      id={1}
      tag="a"
    >
      1
    </PaginationLink>
  </PaginationItem>
  <PaginationItem
    disabled={true}
    id="next"
    tag="li"
  >
    <PaginationLink
      id="next"
      next={true}
      tag="a"
    />
  </PaginationItem>
  <PaginationItem
    disabled={true}
    tag="li"
  >
    <PaginationLink
      last={true}
      tag="a"
    />
  </PaginationItem>
</Pagination>
`;
